msteams-platform/build-your-first-app/build-and-run.md

```diff
@@ -1,80 +1,201 @@
+This information is also available in the `README` file of the toolkit.
 
 ## 4. Sideload your app in Teams
 
-Your app is ready to test in Teams. To do this, you must have a Microsoft 365 development account that allows app sideloading. For more information on account opening, see [Teams development account](../build-your-first-app/build-first-app-overview.md#set-up-your-development-account). 
 
-> [!TIP]
-> Check for issues before sideloading your app, using the [validation feature in App Studio](../concepts/deploy-and-publish/appsource/prepare/submission-checklist.md#teams-app-validation-tool), which is included in the toolkit. Fix the errors to successfully sideload the app.
+Sideloading an app is the process of installing apps on your computer, that hasn't gone through the certification process, to appear in the app store and to run on a Windows device. 
 
-Sideload your app in Teams using the following steps:
+The operating system by default blocks you from sideloading apps. You have to enable sideloading of the apps to test your app in Teams. Before you turn on custom app sideloading for your development tenant, you must have one of the following:
 
-> [!NOTE]
-> To enable sideloading before you sideload your app in Teams, follow the steps in [Turn on app sideloading](../concepts/build-and-test/prepare-your-o365-tenant.md#enable-custom-teams-apps-and-turn-on-custom-app-uploading).
+* Ensure to have a Teams account and with admin access.
+* Ensure to have a Microsoft 365 development account that allows app sideloading. 
 
-1. Select the **F5** key to launch a Teams web client in Visual Studio Code.
-1. To display your app content in Teams, specify that where your app is running (`localhost`) is trustworthy:
-   1. Open a new tab in the same browser window (Google Chrome by default) which opened after pressing **F5**.
-   1. Go to `https://localhost:3000/tab` and proceed to the page.
-1. Go back to Teams. In the dialog, select **Add for me** to install your app.
-:::image type="content" source="../assets/images/build-your-first-app/tab-running.png" alt-text="Screenshot showing an example 'Hello, World!' personal tab app running in Teams.":::
+**To enable sideloading of your app in Teams**
 
-🎉 Congratulations! Your app is running in Teams.
+1. Sign in to [Microsoft 365 admin center](https://admin.microsoft.com/Adminportal/Home?source=applauncher#/homepage#/) with your admin credentials. 
+1. Select **Show All** > **Teams**. 
 
-## Next step
+   ![image of admin center menu](~/assets/images/prepare-test-tenant/admin-center.png)
 
-Expand on the personal tab you just created or build another type of Teams app.
+   > [!Note] 
+   > It can take up to 24 hours for the **Teams** option to appear. Meanwhile, you can [Upload your custom app to a Teams environment](/microsoftteams/upload-custom-apps#validate) for testing and validation.
```

Comment:
```suggestion
   > It can take up to 24 hours for the **Teams** option to appear. Meanwhile, you can [Upload your custom app to a Teams environment](/microsoftteams/upload-custom-apps#validate) for testing and validation.
```
```suggestion
   > It can take up to 24 hours for the **Teams** option to appear.
```
Removed the second sentence since it seems to state you can sideload before you've finished enabling sideloading.